### DISCUSSION TITLE
Can the prisma package be moved to devDependencies?

### DISCUSSION THREAD

**Discussion by ryanwoodcox** [epicweb-dev/epic-stack#646] [Q&A]
Is there a reason prisma is installed as a dependency or can and should it be moved to devDependencies?

Supporting documentation:
https://www.prisma.io/docs/getting-started/setup-prisma/add-to-existing-project/relational-databases-typescript-postgresql

**Comment by kentcdodds** [comment]
One benefit is being able to easily run Prisma studio in production (check the database docs). But you do need to make sure it's available for migrations which happen in production so it's really required for us
